### PR TITLE
fix health report upload error

### PR DESCRIPTION
**Pull request by anjalshireesh**
After moving the processing of uploaded health report to background, SUBNET now doesn't return the cluster URL in response of health upload.

This was resulting in error being shown on the UI even when upload to SUBNET was succesful.

Fix this by not expecting the cluster URL in response, and instead linking to the deployments page URL (/health) after succesful upload.

Also start using exposed functions from mc instead of duplicating the logic in console.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -155,3 +155,5 @@ require (
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/minio/mc => github.com/anjalshireesh/mc v0.0.0-20240326105237-ee529683a183
```
Comment (anjalshireesh) [inline go.mod:159]: Will remove this after https://github.com/minio/mc/pull/4885 is merged

**Review by jinapurapu — DISMISSED** [review]

File: web-app/build/index.html [hunk 2]
```diff
@@ -1 +1 @@
...
...,initial-scale=1" name="viewport"/><meta content="#081C42" media="(prefers-color-scheme: light)" name="theme-color"/><meta content="#081C42" media="(prefers-color-scheme: dark)" name="theme-color"/><meta content="MinIO Console" name="description"/><meta name="minio-license" content="agpl"/><link href="./styles/root-styles.css" rel="stylesheet"/><link href="./apple-icon-180x180.png" rel="apple-touch-icon" sizes="180x180"/><link href="./favicon-32x32.png" rel="icon" sizes="32x32" type="image/png"/><link href="./favicon-96x96.png" rel="icon" sizes="96x96" type="image/png"/><link href="./favicon-16x16.png" rel="icon" sizes="16x16" type="image/png"/><link href="./manifest.json" rel="manifest"/><link color="#3a4e54" href="./safari-pinned-tab.svg" rel="mask-icon"/><title>MinIO Console</title><script defer="defer" src="./static/js/main.e7860c5b.js"></script><link href="./static/css/main.e60e4760.css" rel="stylesheet"></head><body><noscript>You need to enable JavaScript to run this app.</noscript><div id="root"><div id="preload"><img src="./images/background.svg"/> <img src="./images/background-wave-orig2.svg"/></div><div id="loader-block"><img src="./Loader.svg"/></div></div></body></html>
```
Comment (prakashsvmx) [inline web-app/build/index.html:1]: This is not required and can be reverted. it would be part of release PR.

File: go.mod (hunk 1 above)
Comment (prakashsvmx) [reply go.mod:159]: @anjalshireesh , could you replace this?

Comment (anjalshireesh) [reply go.mod]: @prakashsvmx done

File: web-app/build/index.html (hunk 2 above)
Comment (anjalshireesh) [reply web-app/build/index.html]: @prakashsvmx thanks, done.

**Review by prakashsvmx — DISMISSED** [review]

**Review by jinapurapu — APPROVED** [review]

**Review by prakashsvmx — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]

**Pull request merged**